modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceLocalServiceImpl.java

```diff
@@ -401,7 +404,7 @@ public DDMFormInstance updateFormInstance(
 
 		Date date = new Date();
 
-		validateFormInstanceSettings(settingsDDMFormValues);
+		validateFormInstanceSettings(settingsDDMFormValues, null);
```

Comment:
@rodrigopaulino does the validation pass if `timeZoneId` is null?

Reply:
This has the same result as the previous call. Meaning, `validateFormInstanceSettings(settingsDDMFormValues)` is equal to `validateFormInstanceSettings(settingsDDMFormValues, null)` and we are not actually using that method in house.